lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb

```diff
@@ -346,7 +346,8 @@ def existing_record_is_nil?
         end
 
         def existing_record_is_blank?
-          @existing_record.present? && existing_value.strip == ''
+          require 'active_support/core_ext/object/blank'
```

Comment:
Can you put this at the top of the file? I think that might be a better place for this.

Also, this should be single quotes, ignore Hound -- I'm not sure what's up, I'll have to take a closer look at that.